chainer_compiler/chainer_compiler.py

```diff
@@ -265,23 +293,35 @@ def compile(self, inputs):
 
         self.compiled = True
 
+        params = dict(self.mc.namedparams())
+        if self.translator == 'onnx_chainer':
+            params = {'param' + key.replace('/', '_'): value for key, value
+                      in params.items()}
+        self.param_values = []
+
+        fwd_chxvm_vars = fwd_graph.params()
+        for name in self.param_names:
+            if name in params:
+                self.param_values.append(params[name])
+            else:
+                out = _resolve_name_correspondence(
+                    self.mc, params, name, self.param_values)
+                if not out:
+                    # Retrieve the initial value from ONNX initializer
+                    if name in fwd_chxvm_vars:
```

Comment:
Do we really need this fallback path?

Reply:
Without this part, I couldn't handle the input value with name such as `MulConstant_0_param_value`.  If we improve onnx_chainer further, we may be able to get rid of this part...?

Reply:
Thanks for your explanation! Now I see the reason. Yeah, I guess onnx-chainer should emit `Constant` instead of initializer for such values. Could you add a TODO I'll comment?